packages/amplication-client/src/Workspaces/hooks/useWorkspaceSelector.ts

```diff
@@ -38,6 +47,23 @@ const useWorkspaceSelector = (authenticated: boolean) => {
     },
   });
 
+  const [
+    getWorkspaces,
+    { loading: loadingWorkspaces, data: workspaceListData },
+  ] = useLazyQuery<workspacesListTData>(GET_WORKSPACES, {
```

Comment:
Workspaces have to taken from context as I remember. Is there a reason to load it again?